app/views/users/new.html.erb

```diff
@@ -1,2 +1,5 @@
-<h1>Users#new</h1>
-<p>Find me in app/views/users/new.html.erb</p>
+<%= form_for @user, url: {controller: "users", action: "create"} do |f| %>
+  <%= f.label "Hi! What's your name?" %>
+  <%= f.text_field :name %>
+  <%= f.submit "Play!" %>
```

Comment:
I was thinking people can just signin in with their name, like in slither.io. then store their name and score in the db. We can even store just the top scorer in the db if we wanted